schemas/v3.1/dialect/base.schema.yaml

```diff
@@ -1,8 +1,8 @@
-$id: https://spec.openapis.org/oas/3.1/dialect/base
+$id: https://spec.openapis.org/oas/3.1/dialect/base/WORK-IN-PROGRESS
```

Comment:
We currently don't publish or reference date-stamped iterations for this file. 

Do we want to change this?

Reply:
@ralfhandl How else would we handle changes to this document?  Changing the valid set of a schema without changing it's `$id` is a problem.